pandas/tests/arrays/test_integer.py

```diff
@@ -934,6 +934,46 @@ def test_astype_nansafe():
         arr.astype("uint32")
 
 
+@pytest.mark.parametrize(
+    "values, expected",
+    [
+        (
+            [9999999999999999, 123123123123123123, 10000000000000543, np.nan],
+            (
+                pd.Series(
+                    IntegerArray(
+                        np.array(
+                            [
+                                9999999999999999,
+                                123123123123123123,
+                                10000000000000543,
+                                0,
+                            ],
+                            dtype="int64",
+                        ),
+                        np.array([False, False, False, True]),
+                    )
+                )
+            ),
```

Comment:
I know it is the Black autoformatter that is creating this verbose output. But since it is only a small parametrization, can you simply put it inside the test to avoid this many lines?

Reply:
I have put that inside the test.